### PR TITLE
Get rid of io module.

### LINKED ISSUE #112: Use fio instead of io
Subj: https://github.com/tarantool/http/blob/master/http/router/fs.lua#L65

### PR DESCRIPTION
**Pull request by Satbek**
In tarantool the standart built-in module for working with files is "fio".
So we should use it instead of "io".

Closes #112

### CODE REVIEW

File: http/router/fs.lua [hunk 1]
```diff
@@ -62,14 +63,14 @@ local function static_file(self, request, format)
         }
     end
 
-    local s, fh = pcall(io.input, file)
+    local fh, _ = fio.open(file, {'O_RDONLY'})
```
Comment (olegrok) [inline http/router/fs.lua:66]: Why don't you handle possible error?

File: http/router/fs.lua [hunk 2]
```diff
@@ -62,14 +63,14 @@ local function static_file(self, request, format)
...
 
-    if not s then
+    if fh == nil then
         return { status = 404 }
     end
 
-    local body = fh:read('*a')
-    io.close(fh)
+    local body = fh:read()
```
Comment (olegrok) [inline http/router/fs.lua:72]: I think we should check errors in all cases and handle them properly. Not just ignore.

File: http/router/fs.lua [hunk 3]
```diff
@@ -62,14 +64,19 @@ local function static_file(self, request, format)
...
+    if err ~= nil then
         return { status = 404 }
     end
 
-    local body = fh:read('*a')
-    io.close(fh)
+    local body
+    body, err = fh:read()
+    if err ~= nil then
+        utils.errorf("Can not return static file for '%s': '%s'", request:path(), err)
```
Comment (olegrok) [inline http/router/fs.lua:76]: After we will return {status = 200, body = nil}. 200 - OK, but we have an error it's not OK.

Comment (Satbek) [reply http/router/fs.lua:75]: No. It's OK. After we will return error response with status = 500.

Comment (olegrok) [reply http/router/fs.lua:75]: Hm...

It's clear above when we `return { status = 404 }`, but completely unclear here

Comment (olegrok) [reply http/router/fs.lua:75]: But anyway it's better than it was before. I mean io to fio change

**Pull request merged**